src/ui/public/utils/decode_geo_hash.js

```diff
@@ -66,11 +72,11 @@ function refine_interval(interval, cd, mask) {
  * @param {number} axis constant for the axis 0=lengthwise (ie. columns, along longitude), 1=heightwise (ie. rows, along latitude).
  * @returns {number} Number of geohash cells (rows or columns) at that precision
  */
-function geohashCells(precision, axis) {
+function geohashCells(precision: number, axis: number) {
   let cells = 1;
   for (let i = 1; i <= precision; i += 1) {
-    //On odd precisions, rows divide by 4 and columns by 8. Vice-versa on even precisions.
-    cells *= (i % 2 === axis) ? 4 : 8;
+    /*On odd precisions, rows divide by 4 and columns by 8. Vice-versa on even precisions */
```

Comment:
nit: I think the first comment was more correct - we generally use `//` for inline comments.